driver/ppm_fillers.c

```diff
@@ -524,6 +524,69 @@ static inline u32 open_flags_to_scap(unsigned long flags)
 	return res;
 }
 
+
+static inline u32 open_modes_to_scap(unsigned long modes)
```

Comment:
This code should follow the kernel style, so tabs to indent code (instead of spaces), spaces around operators (e.g. `res |= PPM_S_IXUSR` instead of `res |=PPM_S_IXUSR`), avoid extra newlines when possible.